### PR TITLE
Crud api

### PR DESCRIPTION
**Pull request by Nguyephi**

### CODE REVIEW

**Review by azdoolittle — APPROVED** [review]
lgtm

File: public/index.html [hunk 1]
```diff
@@ -5,18 +5,9 @@
...
-
   <body>
-
-
-
-
     <div id="reviewsSection"></div>
-
-    <script src="./bundle.js"></script>
+    <script src="./reviews-bundle.js"></script>
```
Comment (kraig-droid) [inline public/index.html:10]: Yay, let's fix all these file naming issues early!

**Review by kraig-droid — APPROVED** [review]
lgtm

File: public/index.html (hunk 1 above)
Comment (Nguyephi) [reply public/index.html:10]: I was inspired by you!

**Review by priscilagonzalez — COMMENTED** [review]
Lgtm.

**Pull request merged**